v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -133,62 +133,45 @@ const populateIdentifiers = (attributeArray, { Config }) => {
 const createPayload = (message, destination) => {
   const { listData } = message.properties;
 
-  if (!listData.add || !isArray(listData.add)) {
-    throw new CustomError(
-      "[Google_adwords_remarketing_list]::add is not present inside listData. Aborting message.",
-      400
+  let outputPayloads = {};
+  const typeOfOperation = Object.keys(listData);
```

Comment:
We can make things cleaner here. Also `typeOfOperation` should be validated against an enum. And we can use switch case to make things clean instead of if else

Reply:
used switch case